azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -459,8 +463,22 @@ private static Account doServicePrincipalLogin(com.microsoft.azure.toolkit.lib.a
         });
     }
 
-    private static void printCredentialDescription(Account account) {
-        System.out.println(account.toString());
+    private static void printCredentialDescription(Account account, boolean skipType) {
+        if (skipType) {
+            if (CollectionUtils.isNotEmpty(account.getSubscriptions())) {
```

Comment:
Duplicate codes with `Account.toString()`, we may add `getSubscriptonDescription` and `getAuthTypeDescription()` later in `Account`

Reply:
will refactor all the usage to account in plugin-lib

Reply:
why not fix to use account.toString() while you at it.

Reply:
because two auth type will be print